composer.json

```diff
@@ -31,6 +27,11 @@
     "phpunit/phpunit": "^9.5",
     "teamneusta/pimcore-testing-framework": "^0.11"
   },
+  "conflict": {
+    "presta/sitemap-bundle": "<3.2",
+    "symfony/form": "<5.4",
+    "symfony/routing": "<5.4"
+  },
```

Comment:
This is necessary so that the pipeline does not fail at the “lowest” dependencies.